src/components/TransactionDetails/TransactionDetailsDrawer.tsx

```
@@ -93,6 +94,28 @@ export const TransactionDetailsReceipt = ({
     const queryClient = useQueryClient()
     const { fetchBalance } = useWallet()
 
+    const [showCancelModal, setShowCancelModal] = useState(false)
+    const handleConfirmCancel = () => {
+        setShowCancelModal(false)
+        setIsLoading && setIsLoading(true)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)